examples/muenster/analysis/test_hdf5.py

```diff
@@ -0,0 +1,6 @@
+__author__ = 'axel'
+import h5py
+import tables
```

Comment:
__[flake8]__
* [E402] module level import not at top of file
* [F401] 'tables' imported but unused